docs/boilerplate/components/Button.md

```diff
@@ -116,6 +124,14 @@ The `pressedStyle` prop is optional. This can be used to style the `Pressable` c
 <Button pressedStyle={{ backgroundColor: "red" }} />
 ```
 
+### `disabledStyle`
+
+The `disabledStyle` prop is optional. This can be used to style the `Pressable` component of the `Button` when the `disabled` prop is truthy. Values passed here will voerride anything set in the preset.
```

Comment:
Small typo here, voerride -> override

Reply:
Whoops! Thank you. Fixed in https://github.com/infinitered/ignite/pull/2593/commits/2336fef1fdfb55c3848ed18349d5b9d94507b9c9